docs/source/api/apollo-server.mdx

```diff
@@ -348,9 +320,11 @@ Controls whether to allow [Batching Queries](../requests/#batching) in a single
 </td>
 <td>
 
-An array of [plugins](../integrations/plugins) to install in your server instance. Each array element can be either a valid plugin object or a zero-argument function that _returns_ a valid plugin object.
+An array of [plugins](/apollo-server/integrations/plugins) to install in your server instance. Each array element can be either a valid plugin object or a zero-argument function that _returns_ a valid plugin object.
 
-In certain cases, Apollo Server installs some of its built-in plugins automatically (for example, when you provide an Apollo Studio API key with the `APOLLO_KEY` environment variable). For details, see the API references for these plugins: [usage reporting](./plugin/usage-reporting/), [schema reporting](./plugin/schema-reporting/), and [inline trace](./plugin/inline-trace/).
+In certain cases, Apollo Server installs some of its built-in plugins automatically (for example, when you provide an Apollo Studio API key with the `APOLLO_KEY` environment variable). For details, see the API references for these plugins: [usage reporting](/apollo-server/plugin/usage-reporting/), [schema reporting](/apollo-server/plugin/schema-reporting/), and [inline trace](/apollo-server/plugin/inline-trace/).
```

Comment:
oh, we also install the default landing page by default

Reply:
Added a link!